geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -99,6 +99,12 @@
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-accumulo-tools_${scala.binary.version}</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-tools_${scala.binary.version}</artifactId>
+            <classifier>data</classifier>
+            <version>${project.version}</version>
```

Comment:
one picky change - add this to the dependencyManagement in the root pom and then remove the version tag here?